cozy/synthesis/core.py

```diff
@@ -42,7 +42,9 @@
 from .enumeration import Enumerator, Fingerprint, eviction_policy
 
 eliminate_vars = Option("eliminate-vars", bool, False)
-enable_blacklist = Option("enable-blacklist", bool, False)
+enable_blacklist = Option("enable-blacklist", bool, False,
+    description='[Experimental] If enabled, skip expressions that has been ' +
+                'found not useful or invalid during improvement searching')
```

Comment:
Every option created via `Option` appears under an "internal options" category when you run Cozy with `--help`. I don't feel that we need the "[Experimental]" tag since _all_ of these options are intended for developers.

Also, small nitpick: "expressions that has been" should be "expressions that **have** been".